### PR TITLE
[ADD] stock_picking_batch_packaging_qty: Packaging in stock picking batch

### PR DESCRIPTION
**Pull request by EmilioPascual**
Added packaging, packaging qty and packaging qty done to views inside batch (tabs operations and detailed operations)

@yajo @Shide @rafaelbn @Gelojr  please review. Thank you.

@moduon MT-5810

### CODE REVIEW

**Review by rafaelbn — APPROVED** [review]
Tested in http://oca-stock-logistics-warehouse-16-0-pr2001-4c10a9778559.runboat.odoo-community.org/web

"Base database", just installed and configured the needed.

👍🏼 

Please @EmilioPascual review my review in loom! 😄 

https://www.loom.com/share/64dc6987c2074f15825bf1701e1d3687?sid=0df658af-aa8f-4e35-a885-25a5d6f11db3

**Comment by rafaelbn** [comment]
Please @yajo review, you wrote ADP MT-5531. 😄 Looks promising!

**Review by Shide — APPROVED** [review]
Code review

File: stock_picking_batch_packaging_qty/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    "category": "Warehouse",
+    "depends": ["stock_picking_batch", "stock_move_packaging_qty"],
+    "data": [
+        "views/stock_move_line_view.xml",
+        "views/stock_move_view.xml",
+    ],
+    "license": "LGPL-3",
+    "installable": True,
+    "autoinstall": True,
+    "maintainers": ["EmilioPascual"],
```
Comment (yajo) [inline stock_picking_batch_packaging_qty/__manifest__.py]:
```suggestion
    "maintainers": ["EmilioPascual", "rafaelbn"],
```

File: stock_picking_batch_packaging_qty/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# Copyright 2021 ForgeFlow S.L. (https://www.forgeflow.com)
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
```
Comment (yajo) [inline stock_picking_batch_packaging_qty/__manifest__.py]: suggestion: please always use https://gitlab.com/moduon/odoo-module-template when creating new modules, so all this boilerplate is correct.

```suggestion
# Copyright 2024 Moduon Team S.L.
# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl-3.0)
```

File: stock_picking_batch_packaging_qty/views/stock_move_view.xml [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+                <field
+                    name="product_packaging_id"
+                    groups="product.group_stock_packaging"
+                    optional="hide"
+                />
+                <field
+                    name="product_packaging_qty"
+                    groups="product.group_stock_packaging"
+                    optional="hide"
+                    attrs="{'readonly': [('product_packaging_id', '=', False)]}"
```
Comment (yajo) [inline stock_picking_batch_packaging_qty/views/stock_move_view.xml]: idea: wouldn't this be better?

```suggestion
                    attrs="{'invisible': [('product_packaging_id', '=', False)]}"
```

**Review by yajo — CHANGES_REQUESTED** [review]

**Review by Gelojr — APPROVED** [review]
LGTM

**Review by yajo — APPROVED** [review]

**Pull request merged**